utils/build-script-impl

```diff
@@ -757,6 +757,9 @@ function set_build_options_for_host() {
         llvm_cmake_options+=(
             -DLLVM_PROFDATA_FILE="${CLANG_PROFILE_INSTR_USE}"
         )
+        swift_cmake_options+=(
+            -DSWIFT_PROFDATA_FILE="${CLANG_PROFILE_INSTR_USE}"
```

Comment:
It's not clear to me that it is always desirable that both the llvm/clang build and the swift build to use the same profile data file, how about using a separate option like `SWIFT_PROFILE_INSTR_USE` so that we have flexibility for this (can either pass the same profile data to both or different data files for each if needed).

Reply:
Good point! Done.

Reply:
Oh, wow, I didn't even see the `CLANG_` prefix there, I read this as `${SWIFT_PROFILE_INSTR_USE}`.  The two definitely need to be different since the paths are going to be pretty different.